On uclibc elf.h does not have GNU extentions but we need this define
so we define it locally if its not getting it from elf.h

Signed-off-by: Khem Raj <raj.khem@gmail.com>

Upstream-Status: Pending

Index: git/elfparser.h
===================================================================
--- git.orig/elfparser.h	2011-07-16 18:57:41.000000000 -0700
+++ git/elfparser.h	2011-07-16 20:28:54.733829895 -0700
@@ -17,6 +17,10 @@
  */
 #include <glib.h>
 
+#ifndef NT_GNU_BUILD_ID
+#define NT_GNU_BUILD_ID 3
+#endif
+
 typedef struct ElfSym ElfSym;
 typedef struct ElfParser ElfParser;
 
